undo commit '[cmake] fix cross compilation' because it requires native tools
https://github.com/FreeRDP/FreeRDP/commit/46bced61fb2f76d71b737349c851b8f9924278d5
diff --git a/client/SDL/common/res/CMakeLists.txt b/client/SDL/common/res/CMakeLists.txt
index fdb80b79b..44edf0553 100644
--- a/client/SDL/common/res/CMakeLists.txt
+++ b/client/SDL/common/res/CMakeLists.txt
@@ -15,16 +15,9 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-if(CMAKE_CROSSCOMPILING)
-   find_package(SdlCommonRes2bin)
-else()
-    add_executable(sdl-common-res2bin
-               convert_res_to_c.cpp
-    )
-   export(TARGETS sdl-common-res2bin FILE
-          "${CMAKE_BINARY_DIR}/SdlCommonRes2binConfig.cmake")
-endif()
-
+add_executable(sdl-common-res2bin
+	   convert_res_to_c.cpp
+)
 
 set(FACTORY_SRCS "")
 set(FACTORY_HDR "")
diff --git a/client/common/man/CMakeLists.txt b/client/common/man/CMakeLists.txt
index a611cc22e..b601f1dfd 100644
--- a/client/common/man/CMakeLists.txt
+++ b/client/common/man/CMakeLists.txt
@@ -1,9 +1,3 @@
-if(CMAKE_CROSSCOMPILING)
-   find_package(GenerateArgumentDocbook)
-else()
-    add_executable(generate_argument_docbook
-        generate_argument_docbook.c
-    )
-   export(TARGETS generate_argument_docbook FILE
-          "${CMAKE_BINARY_DIR}/GenerateArgumentDocbookConfig.cmake")
-endif()
+add_executable(generate_argument_docbook
+    generate_argument_docbook.c
+)
